src/main/java/usecases/UserManager.java

```diff
@@ -58,13 +58,18 @@ public void deleteUser(String username) throws UserNotFoundException {
      * Adds a game that has been played to a User's statistics
      * @param username username for the a User
      * @param win true if the game played was a win, false if not
+     * @param tie true if the game played was a tie, false if not (cannot be true if win is true)
      * @throws UserNotFoundException thrown when a user with a given username does not exist
      */
-    public void addGamesPlayed(String username, boolean win) throws UserNotFoundException {
+    public void addGamesPlayed(String username, boolean win, boolean tie) throws UserNotFoundException {
```

Comment:
Consider using an integer flag? Having these two parameters means that there's a relationship between them. Perhaps using something like -1 for a loss, 0 for a tie, 1 for a win?

Other implementations might work too; mainly just consider using 1 param vs. 2 params.